### ISSUE TITLE
[WebToolsE2E][Aspire][Unstable] After selecting the apiservice console log in "Console", click "Log View" of apiservice and click the "Console" tab, the console log information disappears.

### ISSUE THREAD

**Issue by Rita003** [dotnet/aspire#5691]
**INSTALL STEPS**
1. Clean machine: Win11 x64 23h2 ENU
2. Install SDK 8.0.401
3. Install Aspire 8.2.0

**REPRO STEPS**
1. Open CMD, create an Aspire Starter App using following commands:

```
dotnet new aspire-starter -o AspireStarterApp
cd AspireStarterApp/AspireStarterApp.AppHost
```

2. Dotnet run the project in CMD, and open URL in the browser.
`dotnet run`
3. Click "Console" tab in the left -> Select the apiservice resource -> Click "Resource" tab -> Click "Log View" of apiservice project -> The webpage automatically jumps to apiservice 'Console logs'.
4. Click the "Console" tab again

**Note:**
1. This issue is unstable. If you can't reproduce, you can try creating a new Aspire Starter project and follow the steps to reproduce.

**ACTUAL**
The console log information disappears.
![Console](https://github.com/user-attachments/assets/b874e407-2360-4387-93bf-b31f4dbe00b6)

**EXPECTED**
Console log has some log information

**Comment by davidfowl** [comment]
Unclear if this is a dashboard, app model or orchestrator bug.

cc @JamesNK you dealt with console logs most recently.

**Comment by davidfowl** [comment]
I can't reproduce this on main.

**Comment by Rita003** [comment]
Verify this issue fixed on the latest Aspire 9.0.0-preview.4.24469.3
